libs/sdk-core/src/models.rs

```diff
@@ -1098,6 +1109,23 @@ pub struct Channel {
     ///
     /// This may be empty for older closed channels, if it was not possible to retrieve the closing txid.
     pub closing_txid: Option<String>,
+
+    pub htlc: Option<Vec<Htlc>>,
```

Comment:
I see this field is never None in the code so we can omit the Option as the vector can be empty instead.

Reply:
Done